### PR TITLE
Update LR description

### PR DESCRIPTION
**Pull request by Sussy-OS**
@Botspot Here is the updated discription.

### CODE REVIEW

File: apps/LineRider/description [hunk 1]
```diff
@@ -1,6 +1,11 @@
 An Open Source spiritual successor to the flash game Line Rider
 Line Rider is a classic sandbox game where you draw a track for the sledder to ride on.
 
+Detailed discription:
```
Comment (Crilum) [inline apps/LineRider/description:4]:
```suggestion
Detailed description: 
```
Typo.. :wink:

File: apps/LineRider/description [hunk 2]
```diff
@@ -1,6 +1,11 @@
 An Open Source spiritual successor to the flash game Line Rider
 Line Rider is a classic sandbox game where you draw a track for the sledder to ride on.
 
+Detailed discription: 
+LineRider is an open source flash game where you make tracks using lines.
```
Comment (Crilum) [inline apps/LineRider/description:5]: Does Line Rider still use Flash?

Comment (theofficialgman) [reply apps/LineRider/description:5]: community edition is C#. if you want to write anything about this, @Sussy-OS , you should say that LineRider was a flash game, the community edition is a re-implementation in C#

Comment (Sussy-OS) [reply apps/LineRider/description:5]: Hold on, let me fix that.

**Comment by Sussy-OS** [comment]
Is everyone content with this update?

File: apps/LineRider/description [hunk 3]
```diff
@@ -1,9 +1,10 @@
 An Open Source spiritual successor to the flash game Line Rider
 Line Rider is a classic sandbox game where you draw a track for the sledder to ride on.
 
-Detailed discription: 
-LineRider is an open source flash game where you make tracks using lines. 
-You can animate them, sync them to music, and share them! 
+Detailed description: 
+LineRider was an open source flash game that got re-implemented in C# (LineRider Community Edition).
```
Comment (Crilum) [inline apps/LineRider/description:5]:
```suggestion
LineRider was an open source flash game that got re-implemented in C# (LineRider Community Edition). This is the re-implemented version.
```

**Review by Crilum — CHANGES_REQUESTED** [review]

**Pull request merged**